### DISCUSSION TITLE
Filebeat Azure module configuration

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#5467] [Unsupported Versions] [answered]
Hello, 

I've enabled the filebeat module to collect signin logs from Azure event hubs, but I'm not seeing any data showing in Kibana.  

Relevant filebeat settings from sof-01_sensor.sls
--------
    filebeat:
      third_party_filebeat:
        modules:
          azure:
            signinlogs:
              enabled: true
              var.eventhub: "insights-logs-signinlogs"
              var.consumer_group: "$Default"
              var.connection_string: "Endpoint=sb://obscured.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=obscuredkey"
              var.storage_account: "obscured account"
              var.storage_account_key: "obscuredkey"

I am able to collect logs using a standalone ELK stack with the same azure connection details.  On the standalone stack, filebeat outputs to elasticsearch and on the SO distributed environment filebeat outputs to logstash.  Do I need to enable elasticsearch as an output in the SO environment?

Would someone please be so kind as to kick me toward an understanding of how to get azure log collection working in my distributed SO environment?

**Comment by dougburks** [comment, marked as answer]
Have you checked the Filebeat log for additional clues?
https://docs.securityonion.net/en/2.3/filebeat.html#diagnostic-logging

**Reply by Rbartram** [reply]
Hi Doug.  I had checked and there wasn't anything that I felt was interesting/relevant.  But after a couple days away from SO, I am now seeing azure signin logs.  It would seem that I was just wasn't being patient enough/didn't know what to expect.  

Thank you!
